### PR TITLE
Cybersource payments for FOLIO fines

### PR DESCRIPTION
**Pull request by thatbudakguy**
- Add Cybersource dev doc PDF to README and comments
- Disable payment cookie related methods for Folio and add TODOs
- Implement payment of fines for FOLIO
- Handle the case where Cybersource does not send any unsigned fields

### CODE REVIEW

File: app/models/cybersource/payment_response.rb [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
   # Response sent back by Cybersource after the external checkout is complete
+  # See the PDF linked in the README for more information on the fields
   class PaymentResponse
     class PaymentFailed < StandardError; end
 
     def initialize(fields)
       @fields = fields
       @signed_fields = fields['signed_field_names'].split(',')
-      @unsigned_fields = fields['unsigned_field_names'].split(',')
+      @unsigned_fields = fields['unsigned_field_names']&.split(',') || []
```
Comment (thatbudakguy) [inline app/models/cybersource/payment_response.rb:12]: Cybersource always signs _some_ fields, but may elect to sign _all_ of them, in which case `unsigned_field_names` is apparently not sent. TIL

File: spec/controllers/payments_controller_spec.rb
Comment (thatbudakguy) [inline spec/controllers/payments_controller_spec.rb:1]: hiding whitespace helps with this diff

File: app/models/cybersource/payment_response.rb (hunk 1 above)
Comment (jcoyne) [reply app/models/cybersource/payment_response.rb:12]: That would be a great thing to put in a comment in the code.

Comment (thatbudakguy) [reply app/models/cybersource/payment_response.rb:12]: Added.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**